### PR TITLE
change default kubeconfig location

### PR DESCRIPTION
**Pull request by deads2k**
Changes the kubeconfig location to match kubernetes.

Merge after https://github.com/openshift/origin/pull/3104

### CODE REVIEW

File: cli_reference/get_started_cli.adoc [hunk 1]
```diff
@@ -90,7 +90,7 @@ nicknames.
 As described in the previous section, the `oc login` command automatically
 creates and manages CLI configuration files. All information gathered by the
 command is stored in a configuration file located in
-*_~/.config/openshift/.config_*. The current CLI configuration can be viewed using the following command:
+*_~/.kube/.config_*. The current CLI configuration can be viewed using the following command:
```
Comment (liggitt) [inline cli_reference/get_started_cli.adoc]: `.kube/config`, I think

**Comment by liggitt** [comment]
LGTM other than the one comment

**Comment by deads2k** [comment]
PR merged in origin.  This needs updating nwo.

**Comment by adellape** [comment]
LGTM, thanks.

**Comment by adellape** [comment]
Going to try to get https://github.com/openshift/openshift-docs/pull/467 merged first.

**Pull request merged**